### ISSUE TITLE
MIoU is 乐山市

### ISSUE THREAD

**Issue by chris2000520** [zh320/realtime-semantic-segmentation-pytorch#9]

**Comment by zh320** [comment]
Hi, Chris. Please share the configs for your experiments.

**Comment by chris2000520** [comment]
> Hi, Chris. Please share the configs for your experiments.

I used the same configs as you. I ran 200 rounds of epochs on BiseNetv2, but the best miou is 68.65%. The GPU is RTX 2080 Ti. There are some pictures about my result.
![miou](https://github.com/zh320/realtime-semantic-segmentation-pytorch/assets/51522066/2f6a3b17-9529-4806-8e9e-a641059959d5)
![loss](https://github.com/zh320/realtime-semantic-segmentation-pytorch/assets/51522066/602f2f77-a0ec-4bc1-997d-d4cca35f9422)

**Comment by zh320** [comment]
BiseNetv2 is not pretrained on imagenet dataset, so you need to train it more epochs (e.g. 800 epochs) to achieve better results. Actually you may switch this repo from main branch to archive branch, you will see the mIoU for BiseNetv2 is 69.36 (200 epochs) and 72.15 (800 epochs). Other tricks may increase the performances are: use larger crop-size (e.g. 1024) and use larger batch size. 

Best

**Comment by zhy168** [comment]
> > Hi, Chris. Please share the configs for your experiments.
> 
> I used the same configs as you. I ran 200 rounds of epochs on BiseNetv2, but the best miou is 68.65%. The GPU is RTX 2080 Ti. There are some pictures about my result. ![miou](https://private-user-images.githubusercontent.com/51522066/328758048-2f6a3b17-9529-4806-8e9e-a641059959d5.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0NTIxNDcsIm5iZiI6MTcxOTQ1MTg0NywicGF0aCI6Ii81MTUyMjA2Ni8zMjg3NTgwNDgtMmY2YTNiMTctOTUyOS00ODA2LThlOWUtYTY0MTA1OTk1OWQ1LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI3VDAxMzA0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0NjE2NjU1MjhiMjZmNjcyZjg1NDJjMDYyNDExMTYwMTMwMTI2ZmVhNzQ0OTY4ZTA3ODQ2MmIzZTEwMjI2NmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Hq0QojRXSQlOcwQ4-1wBaBiCsDKKT81wYWDJNAm7bSs) ![loss](https://private-user-images.githubusercontent.com/51522066/328758077-602f2f77-a0ec-4bc1-997d-d4cca35f9422.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0NTIxNDcsIm5iZiI6MTcxOTQ1MTg0NywicGF0aCI6Ii81MTUyMjA2Ni8zMjg3NTgwNzctNjAyZjJmNzctYTBlYy00YmMxLTk5N2QtZDRjY2EzNWY5NDIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI3VDAxMzA0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFkMzU4ZTVhNTU4YTJhM2U5OTM2ZGYyNzdmZDU0OWUxZGRiZGM0NDc1MWEzNjI1Y2NkMWFkZTliNjU3MTYwZDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vo5s50qxfRApskfk6GOa8PvzILK2lGypkFORT9eq_KA)

兄弟你用过这里面STDC的Detail head吗，训练无法收敛

**Comment by chris2000520** [comment]
没有-------- 原始邮件 --------发件人： zhy61718 ***@***.***>日期： 2024年6月27日周四 09:32收件人： zh320/realtime-semantic-segmentation-pytorch ***@***.***>抄送： 书灏 ***@***.***>, Author ***@***.***>主    题： Re: [zh320/realtime-semantic-segmentation-pytorch] MIoU is less  (Issue #9)


Hi, Chris. Please share the configs for your experiments.

I used the same configs as you. I ran 200 rounds of epochs on BiseNetv2, but the best miou is 68.65%. The GPU is RTX 2080 Ti. There are some pictures about my result.  

兄弟你用过这里面STDC的Detail head吗，训练无法收敛
